modules/apps/29-fee/ibc_module.go

```diff
@@ -230,6 +230,13 @@ func (im IBCModule) OnAcknowledgementPacket(
 		return sdkerrors.Wrapf(err, "cannot unmarshal ICS-29 incentivized packet acknowledgement: %v", ack)
 	}
 
+	if im.keeper.IsLocked(ctx) {
+		// if the fee keeper is locked then fee logic should be skipped
+		// this may occur in the presence of a severe bug which leads invalid state
+		// the fee keeper will be unlocked after manual intervention
+		return im.app.OnAcknowledgementPacket(ctx, packet, ack.Result, relayer)
```

Comment:
Let's also document why we're still unmarshalling the acknowledgement because the counterparty is still sending the incentivized ack over